src/registrar/templates/prototype_domain_dns.html

```diff
@@ -14,18 +14,43 @@ <h1>Add DNS records</h1>
     Do note that this just adds records, but does not update or delete existing ones.
     </p>
 
-    <form class="usa-form usa-form--large" method="post" novalidate id="form-container">
-        {% csrf_token %}
-        {% input_with_errors form.name %}
-        {% input_with_errors form.content %}
-        {% input_with_errors form.ttl %}
+    <div x-data="{ show_form: false }">
         <button
-            type="submit"
-            class="usa-button"
+        class="usa-button"
+        type="button"
+        x-on:click="show_form = true"
         >
             Add record
         </button>
-    </form>
+        <!-- Ensures form stays hidden even before Alpine loads -->
```

Comment:
Should this comment specifically go with `x-cloak`? If so it's not clear here above the form (not sure if you can put a comment next to an attribute)